### DISCUSSION TITLE
how to check if three.js is already installed

### DISCUSSION THREAD

**Discussion by emmz3230** [jasonsturges/vite-typescript-npm-package#4] [General]
how to check if three.js is already installed in the dist folder and not install it again if installed already in the project

**Comment by jasonsturges** [comment]
Not sure I'm following your question - if three.js is a peer dependency of your package, it should not be included in your distributable.

Consumers of your package will need to assure they've installed three.js in their projects.

Dependency resolution is a responsibility of applications that use your project.

**Comment by emmz3230** [comment]
bundles three.js but in many cases a site using it will already have their own three.js - is it possible to fix this so it doesn't bundle three.js into the dist bundle?

so this is how the question originally is

**Comment by jasonsturges** [comment]
Your package should not include three.js in the distributable.
